### PR TITLE
[risk=no][RW-4698] rm v3ducc FF references and v2 code from UI

### PR DESCRIPTION
**Pull request by jmthibault79**
Description:

UI portion of RW-4698.  Wait one release cycle for the API portion for rollback safety.

---
**PR checklist**

- [ ] This PR meets the Acceptance Criteria in the JIRA story
- [ ] The JIRA story has been moved to Dev Review
- [x] This PR includes appropriate unit tests
- [x] I have run and tested this change locally
- [ ] I have run the E2E tests on ths change against my local UI and/or API server with `yarn test-local` or [yarn test-local-devup](https://github.com/all-of-us/workbench/blob/master/e2e/README.md#examples)
- [x] If this includes a UI change, I have taken screen recordings or screenshots of the new behavior and notified the PO and UX designer
- [x] If this includes an API change, I have updated the appropriate Swagger definitions and notified API consumers
- [x] If this includes a new feature flag, I have created and linked new JIRA tickets to (a) turn on the feature flag and (b) remove it later

### CODE REVIEW

File: ui/src/app/pages/profile/data-user-code-of-conduct.spec.tsx [hunk 1]
```diff
@@ -37,74 +36,12 @@ describe('DataUserCodeOfConduct', () => {
     profileStore.set({profile, load, reload, updateCache});
   });
 
-  it('should render - v2', () => {
```
Comment (jmthibault79) [inline ui/src/app/pages/profile/data-user-code-of-conduct.spec.tsx:40]: all of these have equivalent v3 tests.  Thanks for making my job easy!

File: ui/src/app/pages/profile/data-user-code-of-conduct.tsx [hunk 2]
```diff
@@ -52,14 +46,38 @@ const styles = reactStyles({
   },
   bold: {
     fontWeight: 600
-  }
+  },
+  textInput: {
```
Comment (jmthibault79) [inline ui/src/app/pages/profile/data-user-code-of-conduct.tsx:50]: moved from data-user-code-of-conduct-styles

File: ui/src/app/pages/profile/data-user-code-of-conduct.tsx [hunk 3]
```diff
@@ -52,14 +46,38 @@ const styles = reactStyles({
...
+    borderRadius: 6
+  },
 });
 
 export enum DataUserCodeOfConductPage {
   CONTENT,
   SIGNATURE
 }
 
+const DuaTextInput = (props) => {
```
Comment (jmthibault79) [inline ui/src/app/pages/profile/data-user-code-of-conduct.tsx:63]: Moved from data-user-code-of-conduct-styles

File: ui/src/app/pages/profile/data-user-code-of-conduct.spec.tsx [hunk 4]
```diff
@@ -115,14 +48,13 @@ describe('DataUserCodeOfConduct', () => {
     expect(wrapper.find('[data-test-id="submit-ducc-button"]').prop('disabled')).toBeTruthy();
 
     // fill required fields
-    wrapper.find('[data-test-id="dua-initials-input"]').forEach((node, index) => {
+    wrapper.find('[data-test-id="ducc-initials-input"]').forEach((node, index) => {
```
Comment (jmthibault79) [inline ui/src/app/pages/profile/data-user-code-of-conduct.spec.tsx:51]: Some opportunistic renames in the spirit of RW-4838.  That's a bigger lift, so I'd rather not go too far in that direction in this PR.

**Review by als364 — APPROVED** [review]

**Pull request merged**